calico-cloud/network-policy/policy-tiers/tiered-policy.mdx

```diff
@@ -62,9 +62,26 @@ The default tier is where:
 - Network and global network policies are placed when you upgrade from Project Calico to $[prodname]
 - Recommended policies are placed when you use the **Recommend a policy** feature
 
-## AdminNetworkPolicy tier:
+## The "adminnetworkpolicy" tier:
```

Comment:
Some questions:

I'm curious about the name format. From what I can tell, we previously used a camel case for this new tier name: AdminNetworkPolicy. Here, you change it to lowercase and add quotes.

The current format is more legible, but makes it look like a resource.

>In addition to AdminNetworkPolicy resources, ... 

Are these proper custom resources? Should they be documented like other custom resources?

Reply:
The actual name of the tier is the all lower case version.  I thought it was misleading to use camel case.  I put it in quotes to emphasise that it's a verbatim name.  It _is_ the name of the resource.  I.e. you can do `kubectl get tier adminnetworkpolicy` to get it; that's what I was trying to emphasize

Reply:
> Are these proper custom resources? Should they be documented like other custom resources?

Yes, they are, we're still deciding on the right treatment for those.  They're documented upstream but I think we should have a page in our docs too.